CMSgov/design-system
# [WNMGDS-2861] Adds Email, Link, RSS & What's New Icon components

## Summary

Adds four new icon components:
- `EmailIcon`
- `LinkIcon`
- `RssIcon`
- `WhatsNewIcon`

See Jira ticket for details: [WNMGDS-2861](https://jira.cms.gov/browse/WNMGDS-2861). Please note that a [Download icon](https://github.com/CMSgov/design-system/blob/main/packages/design-system/src/components/Icons/DownloadIcon.tsx) has already been added to the codebase.

## How to test

1. Run `yarn storybook`.
2. Navigate to the Icons section and verify the presence and appearance of the new icons.

## Checklist


<img width="1035" alt="Screenshot 2024-08-14 at 3 19 39 PM" src="https://github.com/user-attachments/assets/d4b452a8-a92e-412c-9952-76a0a95477c4">
<img width="1032" alt="Screenshot 2024-08-14 at 3 19 57 PM" src="https://github.com/user-attachments/assets/3470a187-84a0-4a57-9195-fdc3a1e5a25f">
<img width="1032" alt="Screenshot 2024-08-14 at 3 20 07 PM" src="https://github.com/user-attachments/assets/0c4b3730-2609-48be-b480-fc839b12d641">
<img width="1032" alt="Screenshot 2024-08-14 at 3 20 23 PM" src="https://github.com/user-attachments/assets/d8ca6ba5-a128-4ff8-9192-a2479e36a03c">
<img width="1035" alt="Screenshot 2024-08-14 at 3 20 49 PM" src="https://github.com/user-attachments/assets/5f7c4d91-dd5a-40d0-9b29-10340b3c71d5">
